3-Authorization-II/1-call-api/AppCreationScripts/sample.json

```diff
@@ -1,80 +1,113 @@
 {
   "Sample": {
-    "Title": "Angular single-page application that authenticates users with Azure AD and calls a protected .NET Core web API",
-    "Level": 200,
-    "Client": "Angular SPA",
-    "Service": ".NET Core web API",
-    "RepositoryUrl": "ms-identity-javascript-angular-tutorial",
-    "Endpoint": "AAD v2.0"
+      "Author": "derisen",
+      "Title": "Angular single-page application that authenticates users with Azure AD and calls a protected .NET Core web API",
```

Comment:
>Angular single-page application that authenticates users with Azure AD and calls a protected .NET Core web API [](http://example.com/codeflow?start=16&length=110)

An Angular SPA using MSAL Angular to sign-in users with Azure Active Directory and call a protected .NET Core web API